### PR TITLE
speed up tests

### PR DESCRIPTION
**Pull request by barkls**
Mostly just the super slow one, but I also tinkered with a couple others.

### CODE REVIEW

File: holopy/inference/tests/test_nmpfit.py [hunk 1]
```diff
@@ -213,7 +213,7 @@ def test_serialization():
 
 def test_integer_correctness():
     # we keep having bugs where the fitter doesn't
-    schema = detector_grid(shape = 100, spacing = .1)
+    schema = detector_grid(shape = 10, spacing = 1)
```
Comment (briandleahy) [inline holopy/inference/tests/test_nmpfit.py]: Since we're changing this line, can we pep8 it?

**Comment by barkls** [comment]
Test is trivially fast now since it doesn't actually do any sampling. Here we just check that the `sample` function calls `model.sample` with defaults. Actual behaviour of `model.sample` is tested elsewhere.

**Comment by barkls** [comment]
Thanks for fixing that. I'm surprised you can push to my personal branch...

**Comment by briandleahy** [comment]
> Thanks for fixing that. I'm surprised you can push to my personal branch...

BWAHAHAHA :smiling_imp: . 

That's what the "allow edits from maintainers" checkbox. does.

**Comment by barkls** [comment]
did not mean to close...

**Pull request merged**